queries.md

```diff
@@ -2,24 +2,62 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+select * from customers where city = 'London'
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select * from customers where postalcode = 1010
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select phone from suppliers where supplierid = 11
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+select * from orders
+order by orderdate desc
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+select * from suppliers
+where length(supplierName) > 20
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+select * from customers where customername like '%market%'
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+insert into customers (customerid, customername, contactname, address, city, postalcode, country)
+values (92, 'The Shire', 'Bilbo Baggins', '1 Hobbit-Hole', 'Bag End', 111, 'Middle Earth')
```

Comment:
since all other columns are specifically named, you actually can get away with not naming customerid or assigning it a vlaue since it will automatically generate an auto-incremented value for you -- however, if you don't call out the specific column names, you _have_ to give it a customerid value of your choice because then it will just be checking how many columns there are (7) and how many values you passed (6)